lib/split/experiment.rb

```diff
@@ -17,6 +19,7 @@ class Experiment
     DEFAULT_OPTIONS = {
       :resettable => true,
       :retain_user_alternatives_after_reset => false,
+      :cohorting_block_magnitude => 1
```

Comment:
instead of block length, I chose to use `cohorting_block_mangitude` which is the number of times each alternative repeats in a single block.

e.g `cohorting_block_magnitude = 3` would produce blocks of length 6 like: [control, alt, control, control, alt, alt]

I did this because since there are two (or more) alternatives some block lengths would be invalid (e.g 5) since the blocks have to be balanced between alternatives